pkg/controller/nexus/resource/persistence/manager.go

```diff
@@ -41,11 +44,20 @@ type manager struct {
 }
 
 // NewManager creates a persistence resources manager
-func NewManager(nexus *v1alpha1.Nexus, client client.Client) *manager {
-	return &manager{
-		nexus:  nexus,
+func NewManager(nexus v1alpha1.Nexus, client client.Client) *manager {
```

Comment:
`manager` is implementing a public `interface`? I believe we should coordinate with #88

Reply:
Actually, `NewManager` must return `Manager` interface